### PR TITLE
#RI-5118 - update readme

### PR DESCRIPTION
**Pull request by romansergeenkosofteq**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,3 +202,33 @@ redis:[results=single;pipeline=4]
 redis:[results=group;mode=ascii]
 redis:[auto=true;pipeline=8;mode=raw;results=single;]
 ```
```
Comment (ViktarStarastsenka) [inline README.md:204]: will `auto=false` still work?

File: README.md [hunk 2]
```diff
@@ -76,24 +77,25 @@ If you would like to customize or prettify the archive name, page names, or the
 Manifest file should start with single Node and should have properties described
```
Comment (ViktarStarastsenka) [inline README.md:77]: let's update the file with examples

File: README.md [hunk 3]
```diff
@@ -76,24 +77,25 @@ If you would like to customize or prettify the archive name, page names, or the
...
-| description             | string                   | Not required, short description of tutorials                                                                                                |
+| Prop                    | Type     | Description                                                         |
+|-------------------------|----------|---------------------------------------------------------------------|
+| label (**required**)    | string   | Label that will be displayed on UI for Node                         |
+| children (**required**) | Node[]   | List of Nodes (nested groups or markdown files)                     |
+| type                    |          | Can be skipped - RedisInsight will always use the "group" parameter |
+| author                  | string   | Not required, Author name                                           |
+| URL                     | string   | Not required, link to repository/web-site                           |
+| industry                | string[] | Not required, tags to show relevant industry                        |
+| description             | string   | Not required, short description of tutorials                        |
```
Comment (ViktarStarastsenka) [inline README.md:89]: please add information on what fields are displayed in RedisInsight and what are not

**Review by ViktarStarastsenka — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,10 +1,11 @@
 ## Introduction
-Tutorials are documents that contain helpful use cases, comments and interesting experience about Redis and Redis Stack capabilities.
-It is a perfect tool to present your use cases and experience using RedisInsight Workbench and share them with others!
 
-This page provides an overview of the Tutorials structure, its elements and contains instructions, recommendations, and best practices for updating and sharing Tutorials.
+Tutorials are interactive documents that contain helpful use cases, comments and interesting experience about Redis to level up your Redis game.
+You can use tutorials already embedded in RedisInsight to explore powerful Redis capabilities, or create your own custom tutorials to present your experience using RedisInsight and share them with others!
```
Comment (dwdougherty) [inline README.md:4]:
```suggestion
Tutorials are interactive documents that contain helpful use cases, comments, and related demonstrations of Redis features to help you level up your Redis game.
You can use RedisInsight's built-in tutorials to explore powerful Redis capabilities or create custom tutorials to present your experience using RedisInsight and share them with others.
```

File: README.md [hunk 5]
```diff
@@ -1,10 +1,11 @@
...
 
-If you want to share your own tutorials - add the `redis-tutorials` label to your GitHub repository and other GitHub users will be able to see and share your experience!
+This page provides an overview of the Tutorials structure and its elements. It also contains instructions, recommendations, and best practices for creating, updating and sharing Tutorials.
+
+If you want to share your own tutorials - add the `redis-tutorials` label to your GitHub repository so other other GitHub users can find and learn your experience.
```
Comment (dwdougherty) [inline README.md:8]:
```suggestion
If you want to share your custom tutorials with others, add the `redis-tutorials` label to your GitHub repository so other GitHub users can find them and learn your experience.
```

File: README.md [hunk 6]
```diff
@@ -23,14 +25,21 @@ Tutorials should be combined into a .zip archive that contains the following:
 2. [manifest.json](#Manifest) that describes the structure and behavior of Tutorials
 
 ## Examples
-If you would like to start with Tutorials, download this example with Redis Stack Tutorials and upload the archive using the "Upload Tutorial" feature in RedisInsight Workbench so you can view and understand the overall logic.
+If you are interested in creating your own tutorials and would like to understand how they work, you can start with the following steps:
+- download this example
+- open RedisInsight
+- navigate to the Insights panel (top right corner)
+- find the "My Tutorials" section on the "Explore" tab
+- click the "Upload Tutorial" button and select the example
```
Comment (dwdougherty) [inline README.md:33]:
```suggestion
- Download this example.
- Open RedisInsight.
- Navigate to the Insights panel (top right corner).
- Find the "My Tutorials" section on the "Explore" tab.
- Click the "Upload Tutorial" button and select the example.
```

File: README.md [hunk 7]
```diff
@@ -23,14 +25,21 @@ Tutorials should be combined into a .zip archive that contains the following:
...
+Your first custom Tutorials are uploaded, and you can open and explore them.
 
-[Redis-Tutorials.zip](https://github.com/RedisInsight/Tutorials/files/11092920/Redis-Tutorials.zip)
+[Tutorials Example](https://github.com/RedisInsight/Tutorials/releases/download/2.42/data.zip)
 
 ## Markdown
 
 In short, Tutorials support the basic [Markdown syntax](https://www.markdownguide.org/basic-syntax/) that you can use to specify your information.
-If you would like to add a button that will insert Redis commands to the Workbench Editor, use Redis Code block that is almost the same as for the [Fenced Code Block](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks), but you need to specify `redis` as language and the label next to it (see the `Create` button example below).
+If you would like to add a button to run commands, use Redis Code block that is almost the same as for the [Fenced Code Block](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks), but you need to specify `redis` as language and the label next to it (see the `Create` button example below).
```
Comment (dwdougherty) [inline README.md:42]:
```suggestion
If you would like to add a button to run commands, use a Redis Code block that is almost the same as a [Fenced Code Block](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks), but you need to specify `redis` as the language and write a label next to it (see the `Create` button example below).
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
Hi Viktar. Just a few copy-edits.

**Review by ViktarStarastsenka — APPROVED** [review]
LGTM

**Review by dwdougherty — APPROVED** [review]

**Pull request merged**